src/nexmo/__init__.py

```diff
@@ -317,7 +317,11 @@ def get_number_insight(self, params=None, **kwargs):
         return self.get(self.api_host, "/number/lookup/json", params or kwargs)
 
     def get_advanced_number_insight(self, params=None, **kwargs):
```

Comment:
should the async vs sync insights call be more explicit? - e.g. have a separate method for async insights - I'm just worried we're adding hidden features that you'd need to know the internal logic of to activate